src/hooks/useWalletSectionsData.ts

```diff
@@ -11,12 +11,14 @@ import { buildBriefWalletSectionsSelector } from '@/helpers/buildWalletSections'
 import { useSortedUserAssets } from '@/resources/assets/useSortedUserAssets';
 import { useLegacyNFTs } from '@/resources/nfts';
 import useNftSort from './useNFTsSortBy';
+import useWalletsWithBalancesAndNames from './useWalletsWithBalancesAndNames';
 
 export default function useWalletSectionsData({
   type,
 }: {
   type?: string;
 } = {}) {
+  const { selectedWallet } = useWallets();
```

Comment:
on line 41, we also use `useWallets` for `isReadOnlyWallet` - can you consolidate and bring `isReadOnlyWallet` up here